src/main.js

```diff
@@ -1,5 +1,10 @@
 import Tag from './Tag.vue'
 
+/**
```

Comment:
I guess it should be above the code

Reply:
I am not an expert on these languages, but sure I can move it up if that is fine - thanks for the feedback 👍

Reply:
I pushed https://github.com/nextcloud/files_automatedtagging/pull/912/commits/b4eaeee1b988e6910229489230a0c8f342902248 (if that is hat you suggested)

Reply:
yes